app/component/itinerary/BicycleLeg.js

```diff
@@ -61,6 +65,16 @@ export default function BicycleLeg(
     networkConfig && networkConfig.type === RentalNetworkType.Scooter;
   const settings = getSettings(config);
   const scooterSettingsOn = settings.scooterNetworks?.length > 0;
+  const LOADSTATE = {
+    UNSET: 'unset',
+    LOADING: 'loading',
```

Comment:
It seems that LOADING is not used anywhere (no spinner rendering), so it can be removed.